test/feeds/feeds.yaml

```diff
@@ -200,6 +200,8 @@ BNB-USD:
 ETH-USD-TWAP-1day:
   discrepancy: 0.5
   precision: 2
+  base: ETH-TWAP
```

Comment:
I know this is only tests but we can not use same base-quote for different values, you are using it for different twaps. 
```suggestion
  base: ETH_TWAP_1d
```

Reply:
done